recipe/meta.yaml

```diff
@@ -29,8 +29,8 @@ requirements:
     - numpy 1.10.*   # [not (win and py>=36)]
     - numpy 1.12.*   # [win and py>=36]
     - six
-    - arrow-cpp 0.9.0
-    - parquet-cpp 1.4.0
+    - arrow-cpp 0.10.0
+    - parquet-cpp 1.4.0.post0
```

Comment:
@cpcloud If I recall correctly We need bump a new parquet-cpp version, am I right?

We also need to wait `arrow-cpp`

Reply:
We have to bump because of ABI changes

Reply:
Sorry, I thought @kszucs meant bump beyond `1.4.0.post0`. We definitely to bump past `1.4.0`.

Reply:
@kszucs Can you update this to be `1.5.0.pre`?

Reply:
Sure, just seen You've merged it

Reply:
I'm waiting with the push until appveyor catches up a bit